packages/backend/src/index.ts

```diff
@@ -55,13 +55,20 @@ import graphql from './plugins/graphql';
 import app from './plugins/app';
 import badges from './plugins/badges';
 import jenkins from './plugins/jenkins';
+import permission from './plugins/permission';
 import { PluginEnvironment } from './types';
+import { ServerPermissionClient } from '@backstage/plugin-permission-node';
 
 function makeCreateEnv(config: Config) {
   const root = getRootLogger();
   const reader = UrlReaders.default({ logger: root, config });
   const discovery = SingleHostDiscovery.fromConfig(config);
-  const tokenManager = ServerTokenManager.noop();
+  const tokenManager = ServerTokenManager.default({ config, logger: root });
```

Comment:
@Rugvip We changed `.fromConfig` to `.default` since we ended up adding the logger (so now it looks more like the `UrlReader` on line 64 above). Let me know if you have thoughts on the name!

Reply:
Ah, the most established pattern we have here is `.fromConfig(config, { ...deps })`. The distinction here would be whether the `config` is used only during instantiation or if it is needed throughout the lifetime of the instance. A check to apply there is whether the config has to be passed on to the constructor or not. I think in this case `.fromConfig(config, { logger })` is the better fit.

The naming of the `UrlReaders.default` is because it gives you the default set of UrlReaders, and it also uses config config object under the hood after instantiation. There is for example also a `UrlReaders.create` where you have to pass in all the UrlReaders yourself.

Reply:
Makes sense, thanks for the context. Changed to `.fromConfig(config, { logger })`.